src/main/java/com/minecolonies/coremod/client/gui/WindowCitizen.java

```diff
@@ -195,10 +247,85 @@
      */
     private static final String WINDOW_ID_SATURATION_BAR = "saturationBar";
 
+    /**
+     * Id of the gender button.
+     */
+    private static final String WINDOW_ID_GENDER = "gender";
+
+    /**
+     * Requests list id.
+     */
+    private static final String WINDOW_ID_LIST_REQUESTS = "requests";
+
+    /**
+     * Requestst stack id.
+     */
+    private static final String LIST_ELEMENT_ID_REQUEST_STACK = "requestStack";
+
+    /**
+     * Resolver string.
+     */
+    private static final String DELIVERY_IMAGE = "deliveryImage";
+
+    /**
+     * Button id of the requests page.
+     */
+    private static final String BUTTON_REQUESTS = "requestsTitle";
+
+    /**
+     * Button id to get back from the requests page.
+     */
+    private static final String BUTTON_BACK = "back";
+
+    /**
+     * Id of the pages view.
+     */
+    private static final String VIEW_PAGES = "pages";
+
+    /**
+     * Source of the female wax location.
+     */
+    private static final String FEMALE_SOURCE = "minecolonies:textures/gui/citizen/colonist_wax_female_smaller.png";
+
+    /**
+     * Id of the detail button
+     */
+    private static final String REQUEST_DETAIL = "detail";
+
+    /**
+     * Id of the short detail label.
+     */
+    private static final String REQUEST_SHORT_DETAIL = "shortDetail";
+
+    /**
+     * Id of the requester label.
+     */
+    private static final String REQUESTER = "requester";
+
+    /**
+     * The divider for the life count.
+     */
+    private static final int LIFE_COUNT_DIVIDER = 30;
     /**
      * The citizenData.View object.
      */
     private final CitizenDataView citizen;
+    /**
+     * Scrollinglist of the resources.
+     */
+    private final ScrollingList resourceList;
+    /**
+     * Inventory of the player.
+     */
+    private final InventoryPlayer inventory = this.mc.player.inventory;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1170)